odoo/addons/base/models/ir_actions_report.py

```diff
@@ -689,10 +694,13 @@ def _render_qweb_pdf(self, res_ids=None, data=None):
             data = {}
         data.setdefault('report_type', 'pdf')
 
+        # access the report details with sudo() but evaluation context as sudo(False)
+        SSelf = self.sudo()
```

Comment:
This looks like a type / model. These are generally called something like `self_sudo` or `sudo_self`.